### DISCUSSION TITLE
How to deploy NeMo ASR models at scale?

### DISCUSSION THREAD

**Discussion by asifjahmed** [NVIDIA-NeMo/NeMo#10248] [General]
Hello, I have built a streaming ASR service using NeMo RNNT and CTC models. I am looking for suggestions on how to deploy this in a production environment. I want to implement continuous batching and utilize the GPU to the fullest to support a large number of concurrent requests. Any guidance on this is much appreciated!

**Comment by TechKemon** [comment]
same question here
